compiler/src/iree/compiler/Codegen/LLVMGPU/test/ROCDL/config_user_vector_distribute.mlir

```diff
@@ -71,7 +71,7 @@ hal.executable public @main_0_dispatch_0 {
 
 // CHECK:       #[[$TRANSLATION:.+]] = #iree_codegen.translation_info<LLVMGPUVectorDistribute workgroup_size = [128, 2, 1] subgroup_size = 64
 // CHECK-SAME:    mma_schedule = #iree_gpu.mma_schedule<intrinsic = #iree_gpu.mma_layout<MFMA_F16_16x16x16_F32>,
-// CHECK-SAME:    no_reorder_workgroups
+// CHECK-SAME:    reorder_workgroups = "transpose"
```

Comment:
I don't think this tests what it appears to since the attribute matches the global default. IMO we should maintain a test that disables globally-enables reordering (through the CLI flag) and then add a new test that enabled reordering when disabled globally.